library/src/main/java/com/google/android/material/motion/runtime/TargetScope.java

```diff
@@ -47,6 +47,7 @@
 
   private final SimpleArrayMap<Class<? extends Performer>, Performer> cache =
     new SimpleArrayMap<>();
+  private final SimpleArrayMap<String, Performer.NamedPlanPerformance> namedCache = new SimpleArrayMap<>();
```

Comment:
throughout PR, prefer `import Performer.NamedPlanPerformance` so you can avoid having the use the fully-qualified name in code. Please change here and in other files.